arch/arm/soc/nordic_nrf5/nrf52/arm_mpu_regions.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2017 Linaro Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <arch/arm/cortex_m/mpu/arm_mpu.h>
+
+#include "arm_mpu_mem_cfg.h"
+
+#define FUCR_BASE	0x10000000
+#define APB_PERIPH_BASE	0x40000000
```

Comment:
This looks like there's no tab here.. but in the code there is.  Odd.  (Same below)